### PR TITLE
Add Security Chaos Engineering to the main Glossary - english version

### PR DESCRIPTION
**Pull request by fsbaraglia**

### CODE REVIEW

File: content/en/security_chaos_engineering.md
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:8]: Couple questions:

- Is this practice only applicable to distributed systems?
- Do we need to define chaos engineering first?
- Does this definition also include broader operational concerns or is it only applicable to appsec?

Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:8]: As a note for style purpose, this section should be restricted to defining the practice. How it helps will go in the last section.

File: content/en/security_chaos_engineering.md
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:11]: Please edit this section for clarity. Unless required please limit the problem statement to the one that SCE addresses without mentioning SRE concerns.

File: content/en/security_chaos_engineering.md
Comment (JasonMorgan) [inline content/en/security_chaos_engineering.md:14]: Some of this section belongs in the "what it is" portion. "How it helps" should be limited to describing what SCE does that is beneficial and addresses the problem described in the preceding section.

**Comment by JasonMorgan** [comment]
@fsbaraglia great start but I have a lot of questions. Most are added in line but the main one being should we define chaos engineering first? It seems like this is a subset of that therefore it would benefit from a definition of CE.

I would appreciate your thoughts on this.

Thanks!

**Comment by JasonMorgan** [comment]
Converted to draft while we wait on Chaos Engineering.

**Comment by fsbaraglia** [comment]
Hi @JasonMorgan  , I am finalizing the changes to security chaos engineering, I added in the mean-while the Chaos Engineering concept, if you can please have a look. Thank you

File: content/en/security_chaos_engineering.md
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:8]: - yes it is mainly applicable to distributed systems, for just standalone apps, we can just run the classic regression tests, performance test etc...
- defined and linked 👍 
- yes, it includes infra-sec, app-sec, platform-sec and cloud/hybrid/multi-cloud-security

Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:8]: changes made on this paragraph

File: content/en/security_chaos_engineering.md
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md:11]: done

**Comment by jihoon-seo** [comment]
[Spellcheck CI fails](https://github.com/cncf/glossary/runs/5270751693?check_suite_focus=true).

Adding these words to [`wordlist.txt`](https://github.com/cncf/glossary/blob/main/wordlist.txt) will make Spellcheck CI pass!
- sce
- cyber
- sres

**Comment by JasonMorgan** [comment]
Sorry @fsbaraglia I didn't realize we'd combined these two into one PR. I'll review it now for content but before we merge I'd like each term in it's own PR.

**Review by JasonMorgan — APPROVED** [review]

**Comment by JasonMorgan** [comment]
The terms look great and I'm happy to merge them in. Can you please split Chaos Engineering into it's own PR and appease the spellchecker? This is good stuff and I'm looking forward to getting it merged in.

File: content/en/chaos_engineering.md
Comment (CathPag) [inline content/en/chaos_engineering.md]: Not sure I understand "high deployment of new features..."

File: content/en/security_chaos_engineering.md
Comment (CathPag) [inline content/en/security_chaos_engineering.md:11]: I'm struggling to understand how these sentences connect. The first sentence is clear, but how does "incidents and practices deal both with pre-failure and post-failure situations" fit in and how does it connect to the following sentence. In the last sentence, I added "additionally" because it feels like an extra point. 

I did do some edits but don't think I changed the meaning of the sentences (although I may have by mistake).

File: content/en/security_chaos_engineering.md
Comment (CathPag) [inline content/en/security_chaos_engineering.md:16]: Not sure I get "work against missing **understanding**." Do you mean "unknown security concerns"?

**Review by CathPag — COMMENTED** [review]
Looks great! I made some edits to the field directly. Please have a look to make sure you like them (and I didn't mess anything up).

I do have a few questions. Thank you!

**Comment by fsbaraglia** [comment]
> The terms look great and I'm happy to merge them in. Can you please split Chaos Engineering into it's own PR and appease the spellchecker? This is good stuff and I'm looking forward to getting it merged in.

yes, splitted and created a new PR :D

File: content/en/chaos_engineering.md
Comment (fsbaraglia) [reply content/en/chaos_engineering.md]: added high-frequency deployment

File: content/en/security_chaos_engineering.md
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md]: changed the wording

File: content/en/security_chaos_engineering.md
Comment (fsbaraglia) [reply content/en/security_chaos_engineering.md]: you are right, let me rephrase it, adding "SREs and Cyber Security Engineers" as subject

**Comment by fsbaraglia** [comment]
Hey @JasonMorgan , Hey @CathPag , here the PR for Chaos Eng https://github.com/cncf/glossary/pull/498

**Comment by fsbaraglia** [comment]
nice,  great and thank you all @CathPag  @iamNoah1  @JasonMorgan

**Comment by CathPag** [comment]
The spellcheck is failing. While we wait for Chaos Engineering to be merged, can you please add the missing words, @fsbaraglia?

**Comment by fsbaraglia** [comment]
Sure, let me check if I have the right to add something to wordlist.txt, otherwise I will add to this pull request :)

**Comment by fsbaraglia** [comment]
> The spellcheck is failing. While we wait for Chaos Engineering to be merged, can you please add the missing words, @fsbaraglia?

done, added to the PR the missing words

**Comment by CathPag** [comment]
...and here goes #2 :)

**Pull request merged**
